Shared/Secrets.swift.gyb

```diff
@@ -0,0 +1,37 @@
+// Secrets.swift.gyb
+%{
+import os
+
+def chunks(seq, size):
+    return (seq[i:(i + size)] for i in range(0, len(seq), size))
+
+def encode(string, salt):
+    bytes = string.encode("UTF-8")
+    return [ord(bytes[i]) ^ salt[i % len(salt)] for i in range(0, len(bytes))]
+
+salt = [ord(byte) for byte in os.urandom(64)]
+}%
+enum Secrets {
+
+  static var FeedWranglerClientKey: String {
+      let encoded: [UInt8] = [
+      % for chunk in chunks(encode(os.environ.get('FEED_WRANGLER_KEY'), salt), 8):
+          ${"".join(["0x%02x, " % byte for byte in chunk])}
+      % end
+      ]
+
+      return decode(encoded, salt: salt)
+  }
```

Comment:
Do we have to replicate this whole block every time we need to add a new key?  Could you refactor this guy to have the encoded value generated by a reusable function?

Reply:
This is now generated from a list of secret keys. The array `['MY_KEY', 'MY_OTHER_KEY']` generates blocks for `myKey` and `myOtherKey`

Reply:
Nice work!  That is very cool.